.github/workflows/pr.yml

```diff
@@ -58,12 +58,17 @@ jobs:
           pip install -r requirements.txt -r requirements-dev.txt --no-compile
       - name: start dependencies
         run: docker compose -f docker-compose.dev.yml up -d postgres redis
-      - name: run tests
+      - name: perform database migrations
         run: |
           export HLL_DB_URL=postgresql://${HLL_DB_USER}:${HLL_DB_PASSWORD}@${HLL_DB_HOST}:${HLL_DB_HOST_PORT}/${HLL_DB_NAME}
           export HLL_REDIS_URL=redis://${HLL_REDIS_HOST}:${HLL_REDIS_PORT}/${HLL_REDIS_DB}
           
           PYTHONPATH=. alembic upgrade head
           PYTHONPATH=. python ./manage.py init_db
+          PYTHONPATH=. python ./rconweb/manage.py makemigrations --no-input
```

Comment:
is this actually a good idea? 🤔 I just see we do that in the entrypoint as well, however, I would say that the pipeline and image should only work with the persisted migrations and do not automatically add any new migrations isn't it?
Maybe I misunderstand `makemigrations`, though 🙈

Reply:
Yeah actually; you may be correct since the migrations *should* be (and they are) in source control; but it will also auto detect any changes to the user models we use in Django and run those.

I added that while trying to troubleshoot an issue that was only appearing in tests that I could not duplicate in a real build; I'm going to take it out because in the event we haven't committed a change and it matters I'd like to find out while the tests are running.